### PR TITLE
[MIG][10.0] web_readonly_bypass

### PR DESCRIPTION
**Pull request by simahawk**
replacing https://github.com/OCA/web/pull/451

I fixed the rest of the README and manifest.

### CODE REVIEW

**Comment by simahawk** [comment]
can someone add a "WIP" label? Tests fail
```
2017-10-10 13:17:24,870 5161 INFO ? werkzeug: 127.0.0.1 - - [10/Oct/2017 13:17:24] "GET /web_widget_float_formula/static/tests/js/test_web_widget_float_formula.js HTTP/1.1" 200 -
2017-10-10 13:17:25,235 5161 ERROR openerp_test odoo.tests.common: phantomjs: u"TypeError: undefined is not an object (evaluating 'openerp.testing.section')\n at http://127.0.0.1:8069/web_readonly_bypass/static/test/web_readonly_bypass.js:1 (in global code)\n(leaf frame on top)"
```

**Comment by simahawk** [comment]
@pedrobaeza all green :) No more WIP

**Comment by yvaucher** [comment]
@simahawk also replacing #673 and #601 ?

**Comment by simahawk** [comment]
@yvaucher so far this is the most complete one AFAIK :)
Waiting for reviews tho.

**Comment by yvaucher** [comment]
Added @Ehtaga who created PR #673 as reviewer

File: web_readonly_bypass/views/readonly_bypass.xml [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
+
         <template id="qunit_suite" name="web_readonly_bypass" inherit_id="web.qunit_suite">
-            <xpath expr="//head" position="inside">
+            <xpath expr="//t[@t-set='head']" position="inside">
                 <script type="text/javascript" src="/web_readonly_bypass/static/test/web_readonly_bypass.js"></script>
             </xpath>
         </template>
     </data>
-</openerp>
+</openerp>
```
Comment (Ehtaga) [inline web_readonly_bypass/views/readonly_bypass.xml]: don't remove the new line

**Review by Ehtaga — CHANGES_REQUESTED** [review]
Please cherry pick the 2 commits https://github.com/OCA/web/pull/673/commits/c39506fcac87a2530a0690a7749a63d80db9141f and https://github.com/OCA/web/pull/673/commits/39e60471eadc0004a30111d3b06fcd0ee2cb3572 from the 8.0 version

And look at my PR, the README.rst was more up-to-date, you can add those changes too.

File: web_readonly_bypass/views/readonly_bypass.xml [hunk 2]
```diff
@@ -6,10 +6,11 @@
...
             </xpath>
         </template>
+
         <template id="qunit_suite" name="web_readonly_bypass" inherit_id="web.qunit_suite">
-            <xpath expr="//head" position="inside">
+            <xpath expr="//t[@t-set='head']" position="inside">
                 <script type="text/javascript" src="/web_readonly_bypass/static/test/web_readonly_bypass.js"></script>
             </xpath>
         </template>
     </data>
```
Comment (leemannd) [inline web_readonly_bypass/views/readonly_bypass.xml]: Can you remove `<data>` and s/openerp/odoo/

**Review by leemannd — COMMENTED** [review]
Can't tell about the javascript part.
But I have some nitpicking ;)

**Comment by simahawk** [comment]
@Ehtaga @leemannd hopefully both comments attended

**Review by leemannd — APPROVED** [review]

File: web_readonly_bypass/README.rst [hunk 3]
```diff
@@ -51,7 +51,7 @@ Bug Tracker
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/web/issues/new?body=module:%20web_readonly_bypass%0Aversion:%208.0.1.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`here <https://github.com/OCA/web/issues/new?body=module:%20web_readonly_bypass%0Aversion:%2010.1.0.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (yvaucher) [inline web_readonly_bypass/README.rst]: You can remove this link. We removed it from the template as it was uneasy to maintain.

**Review by yvaucher — APPROVED** [review]
Code review :+1:

**Review by JonathanNEMRY — APPROVED** [review]

**Comment by simahawk** [comment]
ouch, I forgot `[ci skip]` flag... was green anyway! :stuck_out_tongue: 
@Yajo can you review/merge when you have time? Thanks!

**Review by grindtildeath — APPROVED** [review]
Tested and works well, thank you :+1:

**Review by Ehtaga — APPROVED** [review]

**Review by moylop260 — APPROVED** [review]
@pedrobaeza Could you merge it if the commits are done?

**Comment by anhvu-sg** [comment]
Perfect, thank you

**Pull request merged**